src/utils/harborUtils.ts

```diff
@@ -0,0 +1,13 @@
+import { List } from 'immutable';
+import { HarborOption, HarborOptions } from '../types/HarborOptionTypes';
+
+export const getHarbors = (harborsData: any): HarborOptions => {
```

Comment:
is there any reason that the function argument is called `harborsData` instead of `data`?

Reply:
also, shouldn't the data be typed?

Reply:
i dont think this can be typed? GraphQL query is very flexible. As a result, query data structure is also varied base on how much field was from query.

Reply:
The type doesn't need to describe everything inside the data variable. It just needs to describe what this function needs from the data, so it can do what it does.